src/database.cpp

```diff
@@ -211,10 +215,18 @@ std::string Database::open(bool wait, bool memory) {
     sqlite3_free(fail);
 
     if (!wait || ret != SQLITE_BUSY) {
+      if (waiting) {
+        std::cerr << std::endl;
+      }
       close();
       return out;
     } else {
-      std::cerr << "Database wake.db is busy; waiting 1 second ..." << std::endl;
+      if (waiting) {
+        std::cerr << ".";
+      } else {
+        waiting = true;
+        std::cerr << "Database wake.db is busy; waiting 1 second ...";
```

Comment:
```suggestion
        std::cerr << "Database wake.db is busy; waiting.";
```

Reply:
Oh sorry, I missed that you had done the suggestion, I made the change though.